R/create.R

```diff
@@ -240,7 +240,9 @@ create_future_rt <- function(future = c("latest", "project", "estimate"),
 #' stan.
 #' @param rt A list of options as generated by [rt_opts()] defining Rt
 #' estimation. Defaults to [rt_opts()]. Set to `NULL` to switch to using back
-#' calculation rather than generating infections using Rt.
+#' calculation rather than generating infections using Rt. This is the
```

Comment:
I think we decided to move away from back calculation so I'd just say something like

"Set to `NULL` to switch to using the nonmechanistic model instead of the renewal equation model to generate new infections." or something like that.

Reply:
I wanted to change the back calculation wording but there's another issue for that here https://github.com/epiforecasts/EpiNow2/issues/523, which is also dependent on the in-progress PR.

Reply:
> I think we decided to move away from back calculation so I'd just say something like
> 
> "Set to `NULL` to switch to using the nonmechanistic model instead of the renewal equation model to generate new infections." or something like that.

Are you okay with the new text that states the internal settings explicitly?

Reply:
yep

Reply:
See [ac33cf4](https://github.com/epiforecasts/EpiNow2/pull/779/commits/ac33cf4a121135d533e565d3b7c2d8125c519785).